torchok/tasks/pairwise_task.py

```diff
@@ -81,20 +81,20 @@ def forward_with_gt(self, batch: Dict[str, Union[Tensor, int]]) -> Dict[str, Ten
 
     def calc_relevance_matrix(self, y: Tensor) -> Tensor:
         """
-        Calculates binary relevance matrix given multi-label matrice y.
+        Calculates binary relevance matrix given multi-label matrix y.
 
         Args:
             y: Multi-label matrix of shape (N, L) representing labels for N samples, where L - number of classes.
                 Values are either 0 or 1, where y1[i, k] = 1 indicate that i-th sample belongs to k-th class.
+            Values are either 0 or 1, where y1[i, k] = 1 indicate that i-th sample belongs to k-th class.
```

Comment:
Remove row

Reply:
Can you also add documentation for `forward_with_gt` in your format?

Reply:
Done